addons/web_tour/static/src/tour_service/tour_service.js

```diff
@@ -9,16 +9,15 @@ import { session } from "@web/session";
 import { TourPointer } from "../tour_pointer/tour_pointer";
 import { createPointerState } from "./tour_pointer_state";
 import { tourState } from "./tour_state";
-import { callWithUnloadCheck } from "./tour_utils";
 import { TourInteractive } from "./tour_interactive";
 import { TourAutomatic } from "./tour_automatic";
+import { callWithUnloadCheck } from "./tour_utils";
 
 const StepSchema = {
-    id: { type: String, optional: true },
```

Comment:
Important because it's use when patch tour ( add steps in tour with steps.find((step) => step.id === xxx ) )